build-scripts/build_windows_installer.bat

```diff
@@ -45,6 +45,35 @@ if defined VS160COMNTOOLS (
 if defined VS150COMNTOOLS (
   set TOOL_CHAIN_TAG=VS2017
 )
+if defined VS170COMNTOOLS (

```

Comment:
This needs to be moved up to line 41, so that the newest version of Visual Studio takes priority.